### ISSUE TITLE
review 'lymphoproliferative syndrome' and children

### ISSUE THREAD

**Issue by nicolevasilevsky** [monarch-initiative/mondo#6116]
@sabrinatoro this is not high priority, but when you have a chance, could you review the term MONDO:0016537 'lymphoproliferative syndrome'

The xrefs are:
xref: DOID:0060704 {source="MONDO:equivalentTo"}
xref: MESH:D008232 {source="MONDO:equivalentTo"}
xref: NCIT:C9308 {source="MONDO:equivalentTo"}
xref: OMIMPS:308240 {source="MONDO:equivalentTo", source="DOID:0060704"}
xref: Orphanet:238510 {source="MONDO:equivalentTo"}
xref: SCTID:277466009 {source="MONDO:equivalentTo"}
xref: UMLS:C0024314 {source="MONDO:notFoundInDiseaseSubset", source="Orphanet:238510"}
xref: UMLS:CN201619 {source="MONDO:equivalentTo"}

- The [OMIMPS](https://omim.org/phenotypicSeries/PS308240) groups 5 terms and 5 genes together.
- I think the DO term is equiv to the OMIMPS
- I **do not** think the NCIT term is referring to a genetic disease
- The [Orphanet term](https://www.orpha.net/consor/www/cgi-bin/OC_Exp.php?Expert=238510) is a grouping class and references many genes, it doesn't seem equivalent to the OMIMPS, maybe it is equiv to NCIT?
- the [MESH](https://meshb.nlm.nih.gov/record/ui?ui=D008232) seems more general, probably equiv to the NCIT class

(I didn't check SCTID or UMLS)

I think these should be split into two terms:
- 'lymphoproliferative syndrome', equiv to DOID, OMIMPS (child of inborn error of immunity)
- 'lymphoproliferative disorder', equiv to MESH, NCIT, Orphanet (child of 'immune deficiency disease')

(and SCTID and UMLS mappings are TBD)

If you agree, we should probably add comments to both terms to note that they are intentionally split, even though the names are similar (because I could see myself getting confused by that in the future).

(and if you agree with this, we should review the children.)

I can do this work, I would just like your opinion.

Thanks!

**Comment by sabrinatoro** [comment]
@nicolevasilevsky 
I agree with your assessment though it is very unclear to me what the orphanet term refers to (it looks like it refers to kind of both? And I see only 1 child term in ols). So I agree, but 
- I would probably leave out the orphanet ID all together (because it has no definition, it is not clear what it is referring to, it has no mapping on its own, and have only 1 child). maybe add as "related" to both term? 
- let's review the definition for both terms to make sure the difference between the terms are
- let's review all the synonyms. 

I hope that helps. 

And thank you for adding all the links to the x-ref! That made everything easier. :-)
